ONSdigital/eq-terraform
# [EQ-273] Split subnets for waf and better network segmentation.

**What**

This PR re-organises the VPC ip subnets to create three distinct IP ranges for use by our WAF, Application and associated Tooling. 

**How to test**
1. Deploy a new dev environment having set the correct tfvars for the three subnet CIDR's
2. Check that the deployment has completed ok and that the healthcheck works.
3. Check that you can't reach any of the non-public parts of the system (Rabbitmq servers, Jenkins subnet)

**Who can test**

Anyone but @dhilton
